papis/commands/rm.py

```diff
@@ -146,15 +146,14 @@ def cli(query: str,
     if not (_file or _notes):
         for document in documents:
             if not force:
-                tbar = "The folder {0} would be removed".format(
-                    document.get_main_folder())
-                logger.warning("This document will be removed, check it")
+                logger.warning("Folder to be removed: {0}".format(
+                               document.get_main_folder()))
```

Comment:
```suggestion
                logger.warning("Folder to be removed: '%s'", 
                               document.get_main_folder())
```
? Logging generally uses the `%` formatting.